### DISCUSSION TITLE
Integration with iOS Shortcuts/Automation

### DISCUSSION THREAD

**Discussion by ddman2** [LeoKlaus/plappa#126] [Feature Requests]
_**### Feature Request: Integration with iOS Shortcuts/Automation**_

**Summary**
It would be great to have integration of Plappa with iOS Shortcuts/Automation to enable users to automate tasks such as starting the playback of the current audiobook.

**Description**
Please provide support for iOS Shortcuts by exposing actions that can be used within the Shortcuts app. Specifically, the ability to:

Start playing the current audiobook.
Pause playback.
Skip to the next chapter.

These actions will allow users to create custom automation, such as starting audiobook playback when connecting to CarPlay.

For more information on how to integrate with the Shortcuts app, please refer to Apple's [Shortcuts API documentation](https://developer.apple.com/documentation/sirikit).

**Implementation**
Expose Actions: Add actions like "Start Current Audiobook," "Pause Playback," and "Skip Chapter" to the iOS Shortcuts app.

Thank you for considering this feature request.

**Comment by LeoKlaus** [comment]
I'll close this now with #125 
The update is currently live in the experimental TestFlight branch (https://testflight.apple.com/join/oSAekRD5) and should hit the regular TestFlight channel and AppStore fairly soon.
